php/Terminus/Models/Collections/TerminusCollection.php

```diff
@@ -89,7 +89,7 @@ public function ids() {
    *
    * @param [stdClass] $model_data Data to feed into attributes of new model
    * @param [array]    $options    Data to make properties of the new model
-   * @return [void]
+   * @return [mixed]  $model newly added model
```

Comment:
gotcha. Didn't realize that @return declarations doesn't have to line up with the @param declarations